test/unit/ui/selenium/ViewJobRoles.js

```diff
@@ -0,0 +1,129 @@
+const { resolve } = require("path");
+const { Builder, By } = require('selenium-webdriver');
+const HomePage = require("../../../../src/POM/HomePage");
+const LoginPage = require('../../../../src/POM/LoginPage');
+const { get } = require("http");
+const { assert } = require("console");
+const readline = require('readline');
+const email = process.env.LOGIN_EMAIL;
+const password = process.env.LOGIN_PASSWORD;
+const adminEmail = process.env.LOGIN_EMAIL_ADMIN;
+const adminPassword = process.env.LOGIN_PASSWORD_ADMIN;
+require('dotenv').config();
+
```

Comment:
This file should be in test/ui/selenium/ViewJobRoles

Reply:
done